docs/stale-subvolume.md

```diff
@@ -0,0 +1,38 @@
+# subvolume cleanup
+
+The subvolume command is used to clean the stale subvolume/images
+which has no parent-pvc attached to it.
+The command would list out all such subvolume/images which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+Few cases are mentioned below:
+* Multiple filesystems in the ceph cluster
+* Multiple subvolume groups in the same filesystem
+* Subvolume with retain-snapshot feature (only snapshot exists on this subvolume)
+* Pending clones (PVC)
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the stale subvolume/images that can be removed
+* `clean` : [clean](#clean) stale subvolume/images after user's confirmation
+
+## ls
+
+```bash
+kubectl rook-ceph subvolume ls
+
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110004
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110005
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110006
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110007
+
+```
+
+## clean
+
+```bash
+kubectl rook-ceph subvolume clean csi-vol-427774b4-340b-11ed-8d66-0242ac110004
```

Comment:
"Cleaning" a volume is just deleting it, right? For clarity, IMO we should call it "delete".
```suggestion
kubectl rook-ceph subvolume delete csi-vol-427774b4-340b-11ed-8d66-0242ac110004
```

Reply:
yes, we can do that.